tests/mypy/outputs/plugin-fail-strict.txt

```diff
@@ -36,9 +36,8 @@
 219: error: Property "y" defined in "FrozenModel" is read-only  [misc]
 240: error: Incompatible types in assignment (expression has type "None", variable has type "int")  [assignment]
 241: error: Incompatible types in assignment (expression has type "None", variable has type "int")  [assignment]
-244: error: Incompatible types in assignment (expression has type "Set[_T]", variable has type "str")  [assignment]
+244: error: Incompatible types in assignment (expression has type "Set[Any]", variable has type "str")  [assignment]
```

Comment:
Does it make sense to add another entry which checks that a function returning (e.g.) `list[int]` cannot be assigned to `list[str]`?

Reply:
> Sorry again for the bug introduced. :pray:

Really don't worry about it, I create far more bugs than anyone.

Good idea, I'll add.

Reply:
done.